controllers/front/return.php

```diff
@@ -214,27 +214,27 @@ protected function processGetStatus()
 	{
 		header('Content-Type: application/json;charset=UTF-8');
 		/** @var PaymentMethodRepository $paymentMethodRepo */
-		$paymentMethodRepo = $this->module->getContainer(PaymentMethodRepository::class);
+		$paymentMethodRepo = $this->module->getMollieContainer(PaymentMethodRepository::class);
 
 		$transactionId = Tools::getValue('transaction_id');
 		$dbPayment = $paymentMethodRepo->getPaymentBy('transaction_id', $transactionId);
 		$cart = new Cart($dbPayment['cart_id']);
 		if (!Validate::isLoadedObject($cart)) {
-			die(json_encode([
+			exit(json_encode([
```

Comment:
whats the argument for this change ?

Reply:
Unsure why merging changed this, will fix it in other places aswell.

Reply:
maybe its cs fixer rule or something then  - anyway, need to understand then why its better to use this rather then die.